scripts/kconfig/kconfigfunctions.py

```diff
@@ -59,9 +69,15 @@ def dt_get_hex_val(kconf, name, arg, unit=None):
         d = 0x0
     return str(d)
 
+def dt_dev_exist_by_path(kconf, name, dts_path):
+    if dts_path in reduced.keys():
```

Comment:
Just `dts_path in reduced` would work too. `in` with a dictionary checks the keys. Something like

```python
return "y" if dts_path in reduces else "n"
```